README.md

```diff
@@ -59,6 +60,31 @@ Import sonatype-visualstudio-settings.xml using Tools -> Import and Export Setti
 
 Import sonatype-visualstudio-resharper-settings.xml using ReSharper -> Options... -> Manage... -> Import and Export -> Import from file...
 
+### VSCode

```

Comment:
might be worth mentioning here that VS Code uses the Eclipse formatter to format the code

Reply:
I have added this note. It is plain text editor until user installs a Java Extension Pack :)